packages/components/src/popover/stories/index.js

```diff
@@ -244,7 +244,7 @@ export const WithSlotOutsideIframe = ( args ) => {
 				<Iframe
 					style={ {
 						width: '100%',
-						height: '100%',
+						height: '400px',
```

Comment:
These are just some cosmetic changes for the Storybook iframe example (see PR description for more info)